third_party/move/move-binary-format/src/deserializer.rs

```diff
@@ -942,6 +974,87 @@ fn load_signature_tokens(cursor: &mut VersionedCursor) -> BinaryLoaderResult<Vec
     Ok(tokens)
 }
 
+fn load_access_specifiers(
+    cursor: &mut VersionedCursor,
+) -> BinaryLoaderResult<Option<Vec<AccessSpecifier>>> {
+    load_option(cursor, |cursor| {
+        let mut specs: Vec<AccessSpecifier> = vec![];
+        for _ in 0..load_access_specifier_count(cursor)? {
+            specs.push(load_access_specifier(cursor)?)
+        }
+        Ok(specs)
+    })
+}
+
+fn load_access_specifier(cursor: &mut VersionedCursor) -> BinaryLoaderResult<AccessSpecifier> {
+    let b = load_u8(cursor)?;
+    let kind = if b == SerializedAccessKind::READ as u8 {
```

Comment:
See various `from_u8()` functions used in that same file to allow match.  That's another existing pattern.

Reply:
Ok, I did not pay attention to this, done the same way now. Rather ugly though because you have the raw codes now two times, once in the definition of the enum, and again in the `from_u8`...